activation/activation.go

```diff
@@ -165,6 +167,7 @@ func NewBuilder(
 		coinbaseAccount:       conf.CoinbaseAccount,
 		goldenATXID:           conf.GoldenATXID,
 		layersPerEpoch:        conf.LayersPerEpoch,
+		regossipInterval:      conf.RegossipInterval,
```

Comment:
Would it make sense to set a minimum interval to avoid spamming when bad config is applied? Like:
```suggestion
		regossipInterval:      max(time.Second * 30, conf.RegossipInterval),
```

Reply:
libp2p also maintains a cache with sent/received message. so spamming Publish won't spam neighbours. i will check how long it keeps that in memory

Reply:
> 	// TimeCacheDuration specifies how long a message ID will be remembered as seen.
	// Use WithSeenMessagesTTL to configure this per pubsub instance, instead of overriding the global default.
	TimeCacheDuration = 120 * time.Second